src/utils/ExtensionUtilities.js

```diff
@@ -17,7 +18,7 @@ export function acceleratedRaycast( raycaster, intersects ) {
 		tmpInverseMatrix.copy( this.matrixWorld ).invert();
 		ray.copy( raycaster.ray ).applyMatrix4( tmpInverseMatrix );
 
-		this.getWorldScale( worldScale );
+		getWorldScale( this.matrixWorld, worldScale );
```

Comment:
Oh interesting... So the previous issue was caused by `Object3D.getWorldScale` [implicitly updating the world matrix](https://github.com/mrdoob/three.js/blob/134ff886792734a75c0a9b30aa816d19270f8526/src/core/Object3D.js#L512-L520) without having synchronized position, rotation, and scale values. Confusing.

Why don't we use `Matrix4.decompose`, instead, and pass a dummy position and rotation into the function? I think it will be simpler than copying logic and directly accessing matrix elements.